service/src/test/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelperTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JudgeAndLegalAdvisorHelperTest {
+    @Test
+    public void shouldHaveEmptyLegalAdvisorNameWhenJudgeAndLegalAdvisorNull() {
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = null;
+
+        String legalAdvisorName = JudgeAndLegalAdvisorHelper.getLegalAdvisorName(judgeAndLegalAdvisor);
+
+        assertThat(legalAdvisorName).isEqualTo("");
+    }
+
+    @Test
+    public void shouldHaveEmptyJudgeNameWhenJudgeAndLegalAdvisorNull() {
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = null;
+
+        String judgeTitleAndName = JudgeAndLegalAdvisorHelper.formatJudgeTitleAndName(judgeAndLegalAdvisor);
+
+        assertThat(judgeTitleAndName).isEqualTo("");
+    }
+
+    @Test
+    public void shouldHaveEmptyJudgeNameWhenJudgeTitleAndNameNotProvided() {
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = JudgeAndLegalAdvisor.builder()
+            .legalAdvisorName("Freddie")
+            .build();
```

Comment:
what are JudgeTitle and JudgeName set to when you don't have them in your builder?

Reply:
☝️

Reply:
null

Reply:
explicitly state that, and then add a test for empty strings :)